### ISSUE TITLE
_tksheets get_dropdowns function throwing TypeError

### ISSUE THREAD

**Issue by AJAY-P-37** [ragardner/tksheet#82]
In this [_tksheet](https://github.com/ragardner/tksheet/blob/master/tksheet/_tksheet.py)  file, line number 890, 

```
 def get_dropdowns(self):
        return {k: v['dropdown'] for k, v in self.MT.cell_options if 'dropdown' in v}
```

1. This function is basically returning a dictionary through dictionary comprehension, but already `self.MT.cell_options` is itself the same dictionary, so this is redundant. Just `return self.MT.cell_options` would be enough.
2. Also while iterating a dictionary through a for loop, in order to get both key and values variables (k, v) in the for loop condition, we should use `dict.items()`, else it will return only the keys from the dictionary.

![image](https://user-images.githubusercontent.com/65544154/135769777-9f057f4a-c497-4f58-aaff-05da135356e1.png)

**Comment by ragardner** [comment]
Hello,

Thanks very much for pointing this out,

The comprehension was indeed missing a `.items()` it should be fixed in `5.0.34`

The dictionary is actually a dictionary of tuples as keys and dictionaries as values though, so in order to make access quicker to the cells with dropdowns I excluded the cell options dictionaries where there were no dropdowns

**Comment by AJAY-P-37** [comment]
That's OK. Thanks for fixing it.
